app/src/main/java/ch/hikemate/app/ui/profile/DeleteAccountScreen.kt

```diff
@@ -0,0 +1,120 @@
+package ch.hikemate.app.ui.profile
+
+import android.app.Activity
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.selection.TextSelectionColors
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.OutlinedTextFieldDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import ch.hikemate.app.R
+import ch.hikemate.app.model.authentication.AuthViewModel
+import ch.hikemate.app.ui.components.BackButton
+import ch.hikemate.app.ui.components.BigButton
+import ch.hikemate.app.ui.components.ButtonType
+import ch.hikemate.app.ui.navigation.NavigationActions
+import ch.hikemate.app.ui.navigation.Route
+import ch.hikemate.app.ui.navigation.Screen
+import ch.hikemate.app.ui.theme.primaryColor
+
+object DeleteAccountScreen {
+  const val TEST_TAG_TITLE = "delete_account_title"
+  const val TEST_TAG_INFO_TEXT = "delete_account_info_text"
+  const val TEST_TAG_PASSWORD_INPUT = "delete_account_password_input"
+  const val TEST_TAG_DELETE_ACCOUNT_BUTTON = "delete_account_delete_account_button"
+}
+
+/**
+ * A composable to display the delete account screen.
+ *
+ * @param navigationActions The navigation actions.
+ * @param authViewModel The authentication view model.
+ */
+@Composable
+fun DeleteAccountScreen(navigationActions: NavigationActions, authViewModel: AuthViewModel) {
+  val context = LocalContext.current
+
+  val passwordMustNotBeEmptyError =
+      stringResource(R.string.delete_account_password_must_be_filled_error)
```

Comment:
Question: Why do you have this string here and all the other ones in the composable tree? Could you maybe normalize it, in the way you want?

Reply:
It is because this string is used inside of a click function which is not in the composable tree. All others are.
Do you think I should put them all in the same place ?